test/views/UpdateYourAddressViewSpec.scala

```diff
@@ -17,8 +17,7 @@
 package views
 
```

Comment:
application stop

Reply:
None of the rest of the view specs have application.stop()